### ISSUE TITLE
Decode mode produces same generated summary for all articles

### ISSUE THREAD

**Issue by theHimanshuShekhar** [theamrzaki/text_summurization_abstractive_methods#23]
If running the model in decode mode using the flag it shows the same generated summary for all the input articles. Any help would be amazing.
![image](https://user-images.githubusercontent.com/31435438/95543077-31248500-0a15-11eb-85fb-7901dc9f23de.png)

**Comment by DengYangyong** [comment]
I meet the sample problem that all the decoded results of different examples are the same no matter I set rl_training=Flase or True, and I have trained the model for about 300000 steps and the loss has stopped failling.

Have you solved the issue?

**Comment by theHimanshuShekhar** [comment]
i went ahead with the pytorch implementation and abandoned trying to fix this. I will leave this issue up in @amrzaki ever looks at this. It might help him fix the issue
